### PR TITLE
add ref data feign client and use it to get the firstname and lastnam…

### PR DESCRIPTION
**Pull request by iguannago**
https://tools.hmcts.net/jira/browse/RWA-319

### CODE REVIEW

File: src/functionalTest/java/uk/gov/hmcts/reform/iacaseapi/CcdScenarioRunnerTest.java [hunk 1]
```diff
@@ -176,6 +176,8 @@ public void scenarios_should_behave_as_specified() throws IOException {
                     .when()
                     .post(requestUri)
                     .then()
+                    .log().ifError()
+                    .log().ifValidationFails()
```
Comment (alexbaeza) [inline src/functionalTest/java/uk/gov/hmcts/reform/iacaseapi/CcdScenarioRunnerTest.java:180]: was this for debug purposes or to improve logging ?

File: charts/ia-case-api/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: ia-case-api
 home: https://github.com/hmcts/ia-case-api
-version: 0.0.20
+version: 0.0.22
```
Comment (alexbaeza) [inline charts/ia-case-api/Chart.yaml:4]: looks like we skipped a version

**Review by alexbaeza — APPROVED** [review]

File: src/functionalTest/java/uk/gov/hmcts/reform/iacaseapi/CcdScenarioRunnerTest.java (hunk 1 above)
Comment (iguannago) [reply src/functionalTest/java/uk/gov/hmcts/reform/iacaseapi/CcdScenarioRunnerTest.java:180]: For both, really. It prints out the error only if there are errors or the validation fails.

File: charts/ia-case-api/Chart.yaml (hunk 2 above)
Comment (iguannago) [reply charts/ia-case-api/Chart.yaml:4]: that's done automatically by Jenkins. I am not sure why it skipped it

Comment (iguannago) [reply charts/ia-case-api/Chart.yaml:4]: Jenkins seems to have bumped it to 21 https://github.com/hmcts/ia-case-api/pull/745/commits/2f3d755f114bd1469ddd9b0aca94dfd758b1a044

**Review by makowalski — APPROVED** [review]
LGTM

**Pull request merged**